test/BenchmarkMeasurementContainerTest.cpp

```diff
@@ -224,7 +218,7 @@ TEST(BenchmarkMeasurementContainerTest, ResultTable) {
             "T", rowNames, columnNames);
 
   // Add measured function to it.
-  table.addMeasurement(0, 1, createWaitLambda(100));
+  table.addMeasurement(0, 1, createWaitLambda(100ms));
```

Comment:
Better also use something shorter than 100ms, the tests run already long enough if they are not stalling because of a wait.

Reply:
Done, see [0fb73ef](https://github.com/ad-freiburg/qlever/pull/1134/commits/0fb73ef2f1539ceec21e1eaa5032db1c22806f40).